README.md

```diff
@@ -1,2 +1,198 @@
 # guzzle-model-client
 PHP wrapper for PHP Guzzle library
+
+Setup
+-----
+
+The recommended way to install guzzle-model-client is through  [`Composer`](http://getcomposer.org). Just create a ``composer.json`` file and run the ``php composer.phar install`` command to install it:
+```json
+{
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/openSILEX/guzzle-model-client"
+        }
+    ],
+    "require": {
+        "openSILEX/guzzle-model-client": "dev-master"
+    }
+}
+```
+
+
+
+## WSJWT (Web service JSON Web Token)
+
+### Key pair authentication
+
+A valid public need to be create key, if you already have a valid public key go to the next step **WSJWT implementation class example**
```

Comment:
"A valid public need to be create key" --> A valid public **key** needs to be created